Update _2020-05-27-practical-tips-for-better-privacy-for-your-busines… (#9)
simpleanalytics/blog: _posts/_2020-05-27-practical-tips-for-better-privacy-for-your-business.md

<img loading="lazy" class="border" style="padding: 1rem; margin: 1rem 0;" src="/images/2020-privacy-tips/requestmap-simpleanalytics.jpg" alt="Third-party scripts of Simple Analytics" />

In the diagram above you only see requests going to servers of Simple Analytics: `simpleanalytics.com`, `simpleanalyticscdn.com`, and `simpleanalytics.io`. The fat pink circle is [our home page video](https://simpleanalytics.com/) which consumes the most bytes. We also use an external payment provider, but we only load the script when the client clicks on the signup, hence in this image you don't see any external scripts.

Compare that to another SaaS like Intercom:

<img loading="lazy" class="border" style="padding: 1rem; margin: 1rem 0;" src="/images/2020-privacy-tips/requestmap-intercom.jpg" alt="Third-party scripts of Intercom" />

> There are many external scripts loaded from all kinds of different domains. I'm not saying this is good or bad, but its a good practice to double check if all those external parties are needed. Run the [Request Map Generator](https://requestmap.webperf.tools/) and see what services you don't need.

Reviewer: ```suggestion
There are many external scripts loaded from all kinds of different domains. I'm not saying this is good or bad, but it's a good practice to double check if all those external parties are needed. Run the [Request Map Generator](https://requestmap.webperf.tools/) and see what services you don't need.
```

> On 29 July 2019, the Court of Justice of the European Union (the "CJEU") ruled that a company embedding on its website a social plugin, such as a Facebook “Like” button, can be considered a data controller [...] – [fieldfisher.com](https://privacylawblog.fieldfisher.com/2019/cjeu-rules-that-companies-using-social-plugins-are-liable-for-the-collection-and-transmission-of-data)

> Although you are not in control of what those platforms behind your widgets do with the data of your customers, you are still responsible for what happens with this information/data.

Reviewer: ```suggestion
Although you are not in control of what those platforms behind your widgets do with the data of your customers, you are still responsible for what happens with this data.
```

1. The internet cables to Iceland are rather slow. Although it’s geographically ideal — located between North America and Europe — in practice, The Netherlands is a faster location for both.
1. We don't want to move to Switzerland because it would be a marketing move only. The EU provides better privacy laws than Switzerland.
1. We need to move anyway because our current provider does not offer the powerful servers we need.

> 1. The second-biggest provider in Iceland is legally headquartered in Hong Kong, which is not a preferred location we want to store our data.

Reviewer: `Honk Kong` haha, nice one

### Don't ask full names

> Unless you really need it. Let people consent to adding their full name. Just ask for a first name if you want to address people in your marketing emails, but don't ask for a full name.

Reviewer: ```suggestion
Unless you really need it. Let people decide for themselves to add their full name. Just ask for a first name if you want to address people in your marketing emails, but don't ask for a full name.
```

### Conclusion

> While working on Simple Analytics, I'm constantly fighting the status quo by finding privacy-friendly ways of handling visitor information. There is way too little guidelines on how to actually prevent tracking in your own business.

Reviewer: ```suggestion
While working on Simple Analytics, I'm constantly fighting the status quo by finding privacy-friendly ways of handling visitor data. There is way too little guidelines on how to actually prevent tracking in your own business.
```
